### PR TITLE
Fix text readability on transaction success page

### LINKED ISSUE #276: Add opacity to TransactionSuccess
The text is a bit hard to read on this page:

https://user-images.githubusercontent.com/8097623/129980482-0f7ed03f-93c4-4585-a5db-d0d2cf728663.mp4

e.g. https://eth2clrfund.netlify.app/#/transaction-success/contribution/0x46a770d89b84d4dc6a62b0f9c6c567509459f6a10b44956500ce6ed83698a5fd

Should we add a layer of opacity so it's easier to read?
Alternatively we could adjust the image so it doesn't overlap with the test (like on the homepage: https://eth2clrfund.netlify.app/#/).

### PR DESCRIPTION
**Pull request by Patil2099**
Fixes #276 
I reduced the opacity of the image-wrapper which makes the text readable. 

image

### CODE REVIEW

**Comment by pettinarip** [comment]
@Patil2099 Thanks for the contribution!!

I like the way you did it. My main concern is whether we need more opacity than `0.8`. I would use `0.6` but let's see what others think.

In either case, I'm ok with it.

File: vue-app/src/views/TransactionSuccess.vue [hunk 1]
```diff
@@ -112,6 +112,7 @@ export default class TransactionSuccess extends Vue {
   width: 100%;
   display: flex;
   justify-content: center;
+  opacity:0.8;
```
Comment (corwintines) [inline vue-app/src/views/TransactionSuccess.vue:115]: Thanks for the contribution @Patil2099!. I think this opacity would be better suited for the .money class. I think the money and confetti which use that class are what is causing this readability issue. If we apply the opacity to them, we can maintain the background this way. I think this also only needs to be done in the desktop size as well.

**Review by corwintines — CHANGES_REQUESTED** [review]

**Comment by corwintines** [comment]
@samajammin since we are adding the gradient to the desktop view for transaction success: https://github.com/ethereum/clrfund/pull/304 was curious if we would still need this.

**Comment by samajammin** [comment]
Hey @Patil2099! Thanks again for getting involved & for the PR. We haven't heard back from you about the requested changes & this issue has since been solved in #304.

Feel free to comment on another issue if you'd like to take a swing at one. Happy to work with you to ensure we can get it merged next time! Cheers.

**Pull request closed without merging**